_includes/navigation.html

```diff
@@ -4,70 +4,70 @@
     {% if site.about_enabled == true %}
       <li>
         <a href="{{ 'about' | relative_url }}" title="About me">
-          <span class="icon icon-about"></span>
+          <svg class="icon-about"><use xlink:href="{% asset_path about.svg %}#icon-about"></use></svg>
```

Comment:
Have tested using `<svg class="icon-rss"><use xlink:href="{% asset_path rss.svg %}#icon-rss" href="{% asset_path rss.svg %}"></use></svg>` and this appears to be unsupported in both Chrome and Firefox latest versions. It does however work in Safari. Likewise, using `href` by itself doesn't work in any browser. Propose continuing to use this method for now so that IE browsers can also be supported with the svgxuse polyfill